### PR TITLE
navi 2.0.0

### PR DESCRIPTION
**Pull request by denisidoro**
The project has been rewritten in Rust. It was previously written in bash.

### CODE REVIEW

**Comment by Bo98** [comment]
> The project has been rewritten in Rust. 

In that case, it needs to be built from source as per https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae.

**Comment by denisidoro** [comment]
Thanks for the response! Tomorrow morning I'm going to rewrite the formula, then. I'm probably use one famous Rust project as reference.

**Comment by chenrui333** [comment]
@denisidoro that works, rust formula is pretty simple and straightforward, be sure to include better tests though. Thanks for your contribution.

File: Formula/navi.rb [hunk 1]
```diff
@@ -11,14 +11,35 @@ class Navi < Formula
     sha256 "64b6dac7d4c689ab6aa5c88ef58369249af2d9a6ac44f56769685569a673dd56" => :high_sierra
   end
 
+  depends_on "rust" => :build
   depends_on "fzf"
 
   def install
-    libexec.install Dir["*"]
-    bin.write_exec_script(libexec/"navi")
+    # copy resources
```
Comment (SMillerDev) [inline Formula/navi.rb]: No need to comment, people familiar with brew/ruby/packaging can just read the code.

**Comment by denisidoro** [comment]
Oops, I force pushed an empty commit and the pull request automatically closed :(

**Pull request closed without merging**